website/docs/api/regions.mdx

```diff
@@ -182,6 +182,189 @@ $ h3 maxPolygonToCellsSize -r 7 -p "[[37.813318999983238, -122.4089866999972145]
+overlapping containment modes.
+
+<Tabs
+  groupId="language"
+  defaultValue="c"
+  values={[
+    {label: 'C', value: 'c'},
+    {label: 'Java', value: 'java'},
+    {label: 'JavaScript (Live)', value: 'javascript'},
+    {label: 'Python', value: 'python'},
+    {label: 'Shell', value: 'shell'},
+  ]
+}>
+<TabItem value="c">
+
+```c
+H3Error polygonToCellsExperimental(const GeoPolygon *geoPolygon, int res, uint32_t flags, int64_t size, H3Index *out);
+```
+
+In C, `polygonToCellsExperimental` takes a `GeoPolygon` struct and preallocated,
+zeroed memory, and fills it with the covering cells.
+
+`size` should be the size in number of `H3Index`s of `out`.
+
+The valid values for `flags` are:
+
+| Enum name | Integer value | Description
+| --------- | ------------- | -----------
+| `CONTAINMENT_CENTER` | 0 | Cell center is contained in the shape
+| `CONTAINMENT_FULL` | 1 | Cell is fully contained in the shape
+| `CONTAINMENT_OVERLAPPING` | 2 | Cell overlaps the shape at any point
+| `CONTAINMENT_OVERLAPPING_BBOX` | 3 | Cell bounding box overlaps shape
+
+Returns 0 (`E_SUCCESS`) on success.
+
+</TabItem>
+<TabItem value="java">
+
+```java
+List<Long> polygonToCellsExperimental(List<LatLng> points, List<List<LatLng>> holes, PolygonToCellsFlags flags, int res);
+List<String> polygonToCellExperimentalAddresses(List<LatLng> points, List<List<LatLng>> holes, PolygonToCellsFlags flags, int res);
+```
+
+The valid values for `flags` are:
+
+| Enum name | Description
+| --------- | -----------
+| `PolygonToCellsFlags.containment_center` | Cell center is contained in the shape
+| `PolygonToCellsFlags.containment_full` | Cell is fully contained in the shape
+| `PolygonToCellsFlags.containment_overlapping` | Cell overlaps the shape at any point
+| `PolygonToCellsFlags.containment_overlapping_bbox` | Cell bounding box overlaps shape
+
+</TabItem>
+<TabItem value="javascript">
+
+```js
+h3.polygonToCellsExperimental(polygon, res, flags, isGeoJson)
+```
+
+```js live
+function example() {
+    const polygon = [
+        [37.813318999983238, -122.4089866999972145],
+        [37.7198061999978478, -122.3544736999993603],
+        [37.8151571999998453, -122.4798767000009008]
+    ];
+    const res = 7;
+    return h3.polygonToCellsExperimental(polygon, h3.POLYGON_TO_CELLS_FLAGS.containmentOverlapping, res);
+}
+```
+
+The valid values for `flags` are:
+
+| Enum name | Description
+| --------- | -----------
+| `POLYGON_TO_CELLS_FLAGS.containment_center` | Cell center is contained in the shape
+| `POLYGON_TO_CELLS_FLAGS.containment_full` | Cell is fully contained in the shape
+| `POLYGON_TO_CELLS_FLAGS.containment_overlapping` | Cell overlaps the shape at any point
+| `POLYGON_TO_CELLS_FLAGS.containment_overlapping_bbox` | Cell bounding box overlaps shape
+
+</TabItem>
+<TabItem value="python">
+
+```py
+h3.polygon_to_cells_experimental(h3shape, res, flags='containment_overlapping')
+h3.h3shape_to_cells_experimental(h3shape, res, flags='containment_overlapping')
```

Comment:
Only need one of these lines?

Reply:
Both are aliases. I do note that in the other bindings I found it more obvious to name the enum PolygonToCellsFlags rather than ContainmentMode, but in Python I copied the ContainmentMode name.